tests/gpuistl/test_gpuPvt.cu

```diff
@@ -0,0 +1,271 @@
+#include <config.h>
+
+#define BOOST_TEST_MODULE TestGpuPvt
+
+#include <boost/test/unit_test.hpp>
+#include <opm/material/common/MathToolbox.hpp>
+#include <opm/material/densead/Evaluation.hpp>
+#include <opm/material/densead/Math.hpp>
+#include <opm/material/common/UniformTabulated2DFunction.hpp>
+#include <opm/material/components/CO2Tables.hpp>
+#include <opm/material/components/CO2.hpp>
+#include <opm/material/components/SimpleHuDuanH2O.hpp>
+#include <opm/material/components/BrineDynamic.hpp>
+#include <opm/material/binarycoefficients/Brine_CO2.hpp>
+#include <opm/material/fluidsystems/blackoilpvt/Co2GasPvt.hpp>
+#include <opm/input/eclipse/EclipseState/Co2StoreConfig.hpp>
+
+#include <opm/simulators/linalg/gpuistl/detail/gpu_safe_call.hpp>
+#include <opm/simulators/linalg/gpuistl/GpuBuffer.hpp>
+#include <opm/simulators/linalg/gpuistl/GpuView.hpp>
+
+#include <cuda_runtime.h>
+#include <vector>
+#include <utility>
+#include <cmath>
+
+using Evaluation = Opm::DenseAd::Evaluation<float, 3>;
+using GpuB = const Opm::gpuistl::GpuBuffer<double>;
+using GpuV = Opm::gpuistl::GpuView<const double>;
+
+using GpuTab = Opm::UniformTabulated2DFunction<double, GpuV>;
+
+using GpuBufCo2Tables = Opm::CO2Tables<double, GpuB>;
+using GpuViewCO2Tables = Opm::CO2Tables<double, GpuV>;
+template Opm::CO2Tables<double, GpuB>::CO2Tables(const Opm::UniformTabulated2DFunction<double, GpuB>&, const Opm::UniformTabulated2DFunction<double, GpuB>&);
```

Comment:
What do these explicit instantiations do? We do not access this in a scope linking to this file.